examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -56,13 +56,14 @@ class CastingPlayerAttributes
     char deviceName[chip::Dnssd::kMaxDeviceNameLen + 1]                    = {};
     char hostName[chip::Dnssd::kHostNameMaxLength + 1]                     = {};
     char instanceName[chip::Dnssd::Commission::kInstanceNameMaxLength + 1] = {};
-    unsigned int numIPs; // number of valid IP addresses
+    unsigned int numIPs; // Number of valid IP addresses
     chip::Inet::IPAddress ipAddresses[chip::Dnssd::CommonResolutionData::kMaxIPAddresses];
     chip::Inet::InterfaceId interfaceId;
     uint16_t port;
     uint16_t productId;
     uint16_t vendorId;
     uint32_t deviceType;
+    uint8_t commissionerPasscode = 0; // Indicates whether a CastingPlayer supports the Commissioner-Generated Passcode feature.
```

Comment:
I am unclear based on the comment how a uint8_t `indicates weather a cstingplayer supports ....`.

Is it `if not 0, ....` ? Maybe we need a constant for this 0 constant to be `kInvalidComissionerPasscode` ? Does the spec explicitly say that 0 is not a valid value or should this be some form of optional?

Reply:
changing to a boolean